README.md

```diff
@@ -11,13 +11,33 @@ Sources to build [EGI documentation static pages](https://egi-foundation.github.
 
 ### Requirements
 
+- mdl
 - hugo
 - NodeJS
   - postcss-cli
   - autoprofixer
 
+### Installation and build
 
-The `.travis.yml` can be used as an example to see how to build the documentation.
+To install npm+nodejs please check the instructions at 
+
+https://www.npmjs.com/get-npm
+
+The rest of the tools can be installed as follows:
+
+```console
+gem install mdl
+npm install postcss-cli
+npm install autoprefixer
+brew install hugo
```

Comment:
So we had various problems due to the hugo version, so now we include it into the repository, it's probably better to use this version as described in the testing locally section. In fact it maybe be better to merge or link those two sections together.

Reply:
please check the new version:-)